apps-rendering/src/components/caption/index.tsx

```diff
@@ -0,0 +1,3 @@
+import Caption from './caption';
```

Comment:
Is there a reason for not having the content of `caption.tsx` inside `index.tsx`?

Reply:
Good question. This is actually part of a larger refactor and I'm anticipating having more components in this folder.